docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit/auto-certificate.md

```diff
@@ -0,0 +1,146 @@
+1. In the **Search** bar, enter `yb.tlsCertificate.organizationName` to view the flag, as per the following illustration:
+
+    ![Custom Organization name](/images/yp/encryption-in-transit/custom-org-name.png)
+
+1. Click **Actions** > **Edit Configuration**, enter a new Config Value, and click **Save**.
+
+## Validate custom organization name
+
+You can verify the organization name by running the following `openssl x509` command:
+
+```sh
+openssl x509 -in ca.crt -text
+```
+
+```output {hl_lines=[6]}
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number: 1683277970271 (0x187eb2f7b5f)
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: CN=yb-dev-sb-ybdemo-univ1~2, O=example.com
+        Validity
+            Not Before: May 5 09:12:50 2023 GMT
+            Not After : May 5 09:12:50 2027 GMT
+```
+
+Notice that default value is `O=example.com`.
+
+After setting the runtime configuration to a value of your choice, (`org-foo` in this example), you should see output similar to the following:
+
+```sh
+openssl x509 -in ca.crt -text -noout
+```
+
+```output
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number: 1689376612248 (0x18956b15f98)
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: CN = yb-dev-sb-ybdemo-univ1~2, O = org-foo
+        Validity
+            Not Before: Jul 14 23:16:52 2023 GMT
+            Not After : Jul 14 23:16:52 2027 GMT
+        Subject: CN = yb-dev-sb-ybdemo-univ1~2, O = org-foo
+        Subject Public Key Info:
+            Public Key Algorithm: rsaEncryption
+                Public-Key: (2048 bit)
+                Modulus:
+```
+
+## Validate certificates
+
+When configuring and using certificates, SSL issues may occasionally arise. You can validate your certificates and keys as follows:
+
+- Verify that the CA CRT and CA private key match by executing the following commands:
+
+    ```shell
+    openssl rsa -noout -modulus -in ca.key | openssl md5
+    openssl x509 -noout -modulus -in ca.crt | openssl md5
+
+    \# outputs should match
+    ```
+
+- Verify that the CA CRT is actually a certificate authority by executing the following command:
+
+    ```shell
+    openssl x509 -text -noout -in ca.crt
+
+    \# Look for fields
+
+    X509v3 Basic Constraints:
+
+      CA:TRUE
+    ```
+
+- Verify that certificates and keys are in PEM format (as opposed to the DER or other format). If these artifacts are not in the PEM format and you require assistance with converting them or identifying the format, consult [Converting certificates](https://support.globalsign.com/ssl/ssl-certificates-installation/converting-certificates-openssl).
+
+- Ensure that the private key does not have a passphrase associated with it. For information on how to identify this condition, see [Decrypt an encrypted SSL RSA private key](https://techjourney.net/how-to-decrypt-an-enrypted-ssl-rsa-private-key-pem-key/).
+
+## Enforce TLS versions
```

Comment:
This section should ideally live outside all these add certificate pages as a separate page. An "Enforce TLS versions" page could live under "enable encryption in transit".